docs/ide4c/vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,19 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/tools/toolutil",
+                "${workspaceFolder}/tools/ctestfw",
+                "${workspaceFolder}/common",
+                "${workspaceFolder}/i18n"
+            ],
+            "defines": [],
```

Comment:
FWIW, I add the following to the defines for my setup:
```
"defines": [
    "U_DISABLE_RENAMING=1",
    "U_COMMON_IMPLEMENTATION"
],
```
The first one helps the "go to definition" functionality avoid confusion with the [urename.h](https://github.com/unicode-org/icu/blob/master/icu4c/source/common/unicode/urename.h) file.
The second one doesn't really do all that much though. (It mostly just makes `UChar` always `char16_t`).

Reply:
+1 for disable renaming.

Reply:
Ooh nice! That (setting U_DISABLE_RENAMING) solves one of my pet peeves with ICU4C development, thanks!

I don't have a good grasp of "U_COMMON_IMPLEMENTATION" yet. What does that do for VSCode, vs U_COMBINED_IMPLEMENTATION or U_STATIC_IMPLEMENTATION? (Does it have something to do with symbol export for debugging? </my best speculation so far>)

Reply:
Given that this json file doesn't permit comments, I can't explain the defines there anyway - but I could mention them in the README.

@jefgen what are the benefits of exporting "library symbols from inside the common library", and the reason to export only those instead of all (U_COMBINED_IMPLEMENTATION)?

The latter being set when U_IN_DOXYGEN is defined... I will see if I can notice a difference when I tweak these defines, but I don't want to include it until I understand it.

Reply:
Hmm.. well taking a second look, TBH, I don't think there is much value in adding `U_COMBINED_IMPLEMENTATION` or `U_COMMON_IMPLEMENTATION`. It might be better to just omit them for now.

Thanks for adding the `U_DISABLE_RENAMING` though! :)